services/director-v2/src/simcore_service_director_v2/core/application.py

```diff
@@ -64,8 +64,7 @@ def init_app(settings: Optional[AppSettings] = None) -> FastAPI:
     if settings.celery.enabled:
         celery.setup(app, settings.celery)
 
-    if settings.registry.enabled:
-        docker_registry.setup(app, settings.registry)
+    dynamic_sidecar.setup(app)
```

Comment:
what about the enable/disable flag there? for consistency?

Reply:
Already tried an had some issues with it. Let it like this. Maybe I will come back when I've processed all the feedback from the reviews.

Reply:
outdated